src/NUnitFramework/Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <LangVersion Condition="'$(MSBuildProjectExtension)' == '.csproj'">7</LangVersion>
+    <LangVersion Condition="'$(MSBuildProjectExtension)' == '.csproj'">8</LangVersion>
```

Comment:
I'm probably not a good reviewer of the nullability stuff, but this change caught my eye. Would this require VS2019 as a build tool? If so, we should update BUILDING.md, as it still calls out VS2017 as a req.

Reply:
Yes, thank you!